src/locales/ar.po

```diff
@@ -1221,10 +1221,10 @@ msgid "Website exclusion is currently set to <strong>%1.</strong>"
 msgstr "استبعاد الموقع مضبوط حالياً على <strong>%1.</strong>"
 
 msgid "Win payout if %3 after %plural(%5,%d tick,%d ticks) is higher than or equal to %6."
-msgstr "أربح عائد إذا %3 بعد % الجمع(%5, علامة دي%, علامات دي%) أعلى من أو تساوي ل %6."
+msgstr "أربح عائد إذا %3 بعد %plural(%5,%d علامة,%d علامات) أعلى من أو تساوي ل %6."
```

Comment:
That is buggy for 2 reasons:

1. Arabic has 6 plural forms. It looks like there are only 2 provided.
2. where is the closing `)` for `%plural(`?